internal/handlers/jsonb1/where.go

```diff
@@ -68,6 +72,18 @@ func scalar(v any, p *pg.Placeholder) (sql string, args []any, err error) {
 	return
 }
 
+func composite(v any, p *pg.Placeholder) (sql string, args []any, err error) {
```

Comment:
Please add documentation comment

Reply:
The func was refactored. Documentation to the refactored func was added